### DISCUSSION TITLE
Interesting patches in forks

### DISCUSSION THREAD

**Discussion by nittka** [TVTower/TVTower#324] [General]
I briefly went through the TVTower-Forks. Some of them contain branches with patches that might be interesting for merging (localization, fixes for database entries...). What would be a good way to contact the owners to ask if they are interested?
* @DerDannyF programmes.xml
* @Nalindir typos etc.
* @XtrmPrgrmmr localization
* @zforni localization
* @olsza localization (new language)

**Comment by GWRon** [comment]
Did not know about the last one.

Danny..think I could mail him.
Nalindir...hmm became quite quiet in the discord channel.

The Turkish xtrmprogrammer..could mail him too I think.

**Comment by XtrmPrgrmmr** [comment]
Hi,
That's right. 
I translated in Turkish, but didn't checked for 2 (maybe 3 - 4 - ? :) ) years...

I agree, what you want. (But if you need, I may also update files? )
Just let me know your decision.

**Comment by GWRon** [comment]
nittka redesigned translation files a bit. So you can translate "parts" more easily.

If you find the time...just check it out. I am sure you fiddle out without any further needed help. But if you need some...we stand by ready to assist.

**Reply by XtrmPrgrmmr** [reply]
Thanks... I may find time this weekend and will try. (If need, I'll call. ;) )

**Comment by nittka** [comment]
@starkjoss created a fork starting patches for Spanish translations... (If you add new translations where there were none before - these are marked by `#` - remove that comment character for the translation to take effect)
